server/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaRequestingExecutionTokenSpec.scala

```diff
@@ -27,7 +27,7 @@ class EjeaRequestingExecutionTokenSpec extends EngineJobExecutionActorSpec with
 
     CallCachingModes foreach { mode =>
       s"check against the Job Store if restarting is true ($mode)" in {
-        ejea = helper.buildEJEA(restarting = true)
+        ejea = helper.buildEJEA(restarting = true)(RequestingRestartCheckToken)
```

Comment:
this should probably now be in a new `EjeaRequestingRestartTokenSpec` since that is now a distinct state... though it doesn't seem to be doing too much harm here